docs/Reference/JSON-RPC-API-Methods.md

```diff
@@ -817,20 +817,18 @@ You can interact with contracts using [eth_sendRawTransaction or eth_call](../Us
 
 ### eth_estimateGas
 
-Returns an estimate of how much gas is needed for the transaction to complete. The estimate process does not use
-gas and is not added to the blockchain as a transaction. The estimate can be greater than the amount of gas that the
-transaction actually uses, for reasons including EVM mechanics and node performance.
+Returns an estimate of how much gas is needed for a future transaction to complete. The estimation process does not use
+gas and is not added to the blockchain as a transaction. Setting a gas limit is irrelevant to the estimation process
```

Comment:
'and is not added' -> 'and the transaction is not added' 

I think it would be clearer to move the part about setting a gas limit having no effect the the parameters section and say specifically something like 'The `gas` property on the transaction all object is ignored by `eth_estimateGas`